micromagneticmodel/driver.py

```diff
@@ -138,9 +139,12 @@ def drive(
                 ovf_format=ovf_format,
                 **kwargs,
             )
+            self._start_time = datetime.datetime.now()
+            self._write_info_json(system, **drive_kwargs)
             self._call(system=system, runner=runner, verbose=verbose, **kwargs)
             self._read_data(system)
-
+            self._end_time = datetime.datetime.now()
+            self._update_info_json()
```

Comment:
[I can't propose a change, GitHub does not allow it when lines have been deleted, here is the code]:
```python
try:
    self._call(system=system, runner=runner, verbose=verbose, **kwargs)
except Exception:
    success = False
    raise
else:
    success = True
finally:
    end_time = datetime.datetime.now()
    self._update_info_json(start_time, end_time, success)

self._read_data(system)